byte driver_side
byte driver_dir
byte passenger_side
byte passenger_dir
